mmcv/image/geometric.py

```diff
@@ -37,15 +37,27 @@ def _scale_size(size, scale):
     'lanczos': cv2.INTER_LANCZOS4
 }
 
+# Latest PIL versions use a slightly different naming scheme for filters.
```

Comment:
```suggestion
# Since Pillow v9.1.0, it use a slightly different naming scheme for filters.
```

Reply:
@zhouzaida I have updated the comments with a specific Pillow version

Reply:
Thanks for your contribution.